org.kde.arianna.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "org.kde.arianna",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-22.08",
+    "command": "arianna",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host:ro",
+        "--env=BALOO_DB_PATH=.local/share/baloo",
```

Comment:
it doesn't use the standard directories?

Reply:
It's configured like elisa https://github.com/flathub/org.kde.elisa/blob/master/org.kde.elisa.json#L20